rgw/v2/tests/s3_swift/reusables/server_side_encryption_s3.py

```diff
@@ -44,27 +46,40 @@ def get_object_encryption(s3_client, bucketname, s3_object_name):
     """
     get object encryption for an object, it should be AES256
     """
-    log.info(f"Test if object uploaded is encrypted with AES256")
     get_obj_encryption = s3_client.get_object(Bucket=bucketname, Key=s3_object_name)
     log.info(
         f"server side encryption enabled on {s3_object_name} is : {get_obj_encryption['ServerSideEncryption']}"
     )
```

Comment:
Please remove these lines or add exception code for this line (I would say add these lines in below if..else block)

Reply:
Ahh, right. Addressed.